zulipterminal/ui_tools/boxes.py

```diff
@@ -54,6 +54,9 @@
 from zulipterminal.urwid_types import urwid_Size
 
 
+CONFIRMATION_MSG_LENGTH = 15
```

Comment:
This constant makes it more readable :+1:

Let's leave the value as it is for now - as a constant, it should be easy to adjust later if we need to.

We should probably do a sweep for this with other constants, but marking this as `Final` may help with accidental changing of the value later on.

I suspect we could improve the name here, avoiding the abbreviation, and indicating if this is a minimum/maximum. Expressive names are better :)

Reply:
The variable has been updated to `MAX_MESSAGE_LENGTH_CONFIRMATION_POPUP`, indicating the maximum message length for which confirmation popup is not prompted. Should I add a comment for further clarity, or is the name itself clear enough?

Or should I change it to `MAX_MESSAGE_LENGTH_NO_CONFIRMATION_POPUP`